src/spicetools/queryutil.py

```diff
@@ -15,22 +17,30 @@ def download_jpl_de(dename="de440s", output=None):
     output : str, pathlib.Path, optional
         Name of the output file path. If not provided, it will be saved into
         this package's `kernels/` directory.
+
+    overwrite : bool, optional
+        If `True`, overwrite the existing file if it exists. Default is
+        `False`, i.e., not download any file but returns the path to the
+        existing one.
     """
     if not dename.endswith(".bsp"):
         dename += ".bsp"
-    url = f"https://ssd.jpl.nasa.gov/ftp/eph/planets/bsp/{dename}"
 
     if output is None:
         # output = os.path.join(os.path.dirname(os.path.abspath(__file__)), "kernels", dename)
         output = Path(__file__).parent / "kernels" / dename
     else:
         output = Path(output)
 
+    if output.exists() and not overwrite:
```

Comment:
Consider adding logging.info or logging.error statements to help track this download.